timemachine/md/thermostat/utils.py

```diff
@@ -1,15 +1,38 @@
+from typing import Optional
+
 import numpy as np
 from numpy.typing import NDArray
 
 from timemachine.constants import BOLTZ
 
 
-def sample_velocities(masses: NDArray, temperature: float) -> NDArray:
-    """Sample Maxwell-Boltzmann velocities ~ N(0, sqrt(kB T / m)"""
+def sample_velocities(masses: NDArray, temperature: float, seed: Optional[int] = None) -> NDArray:
+    """Sample Maxwell-Boltzmann velocities ~ N(0, sqrt(kB T / m)
+
+    Parameters
+    ----------
+
+    masses:
+        Array of masses
+
+    temperature:
+        float representing temperature in kelvin
+
+    seed:
+        integer to use to use as seed, optional. If not provided use global RNG
+
+    Returns
+    -------
+    (N, 3) velocities array, where N is the length of masses
+    """
     n_particles = len(masses)
     spatial_dim = 3
 
-    v_unscaled = np.random.randn(n_particles, spatial_dim)
+    if seed is not None:
+        rng = np.random.default_rng(seed)
+        v_unscaled = rng.standard_normal(size=(n_particles, spatial_dim))
+    else:
+        v_unscaled = np.random.randn(n_particles, spatial_dim)
```

Comment:
Can remove use of global RNG, wasn't sure if this is being relied on else where.